examples/gallery/anaconda-project.yml

```
@@ -22,31 +22,30 @@ variables:
 
 packages: &pkgs
   - python >=3.12
```

```suggestion
  - python >=3.13,<3.14
```
Maybe just pin to a Python version to reduce solve space and causing slow/big solve. 

Not sure about the right syntax